scripts/mono-find-provides.in

```diff
@@ -16,7 +16,7 @@ monolist=($(printf "%s\n" "${filelist[@]}" | egrep "\\.(exe|dll)\$"))
 
 # Only include files with /gac/ in path
 #  (Allows packages to contain private assemblies that don't conflict with other packages)
-monolist=($(printf "%s\n" "${monolist[@]}" | egrep "/gac/"))
+monolist=($(printf "%s\n" "${monolist[@]}" | egrep "/(gac|Facades|4\\.5)/"))
```

Comment:
I guess this includes stuff in `4.5-api` which we don't want, correct?

Reply:
Turns out I read the regex wrong, it'd only match `/gac/`, `/Facades/` or `/4.5/` in the path.